app/models/location.rb

```diff
@@ -56,7 +57,6 @@ class Location < ApplicationRecord
   validates :title, presence: true
   validates :location_type, presence: true, inclusion: { in: location_types }
 
-  scope :supplier, ->(supplier_id) { joins(:suppliers).where(locations_suppliers: { supplier_id: supplier_id }) }
```

Comment:
I think you missed: https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/lib/tasks/notifications.rake#L23

Reply:
Great catch - that rake task is new and wasn't in master when I removed this scope. The scope doesn't work as intended now anyway, so the rake task will need some tweaking. I'll discuss that with Martyn.

Reply:
Discussed and the rake task will be updated to remove use of this method (the method no longer works as intended anyway) prior to running it.

Reply:
👍